checker/jtreg/sortwarnings/OrderOfCheckers.out

```diff
@@ -0,0 +1,19 @@
+OrderOfCheckers.java:15:60: compiler.err.proc.messager: [[all, value]:assignment.type.incompatible] incompatible types in assignment.
+found   : @UnknownVal int @UnknownVal []
```

Comment:
Do you also want to use `-Anomsgtext` to make the output here look like in `ErrorOrders.out`? All the details don't seem necessary.
If the distinction is important, can you document why?